assets/js/components/Site.vue

```diff
@@ -104,6 +108,9 @@ export default {
 		energyflow: function () {
 			return this.collectProps(Energyflow);
 		},
+		loadpointNames: function () {
```

Comment:
loadpointTitles for sake of consistency?

Reply:
yes